### DISCUSSION TITLE
Support for speculative decoding

### DISCUSSION THREAD

**Discussion by BowenBao** [microsoft/onnxruntime-genai#689] [New features / APIs]
Hi team, is there plan to support speculative decoding?

cc @spandantiwari-amd

**Comment by natke** [comment]
Hi @BowenBao, yes we do have speculative decoding on our roadmap but do not have a definite schedule for that yet

**Reply by BowenBao** [reply]
Thank you @natke, we are happy to contribute an initial version :)
